src/desktop/apps/consign/client/actions.js

```diff
@@ -334,7 +334,6 @@ export function handleImageUpload(file) {
         const options = {
           acl: "private",
           app: sd.CONVECTION_GEMINI_APP,
-          key: sd.GEMINI_S3_ACCESS_KEY,
```

Comment:
👋 this is what the changes are like for clients using the correct `@artsy/gemup`